resources/views/objects/search/_forum_post.blade.php

```diff
@@ -14,6 +14,11 @@
     >
         <img class="search-forum-post__avatar-image" src="{{ $user->user_avatar }}">
     </a>
+
+    @if ($post->legacyIcon() !== null)
```

Comment:
I don't think this is a good place to put the icon. It just looks wrong

![](https://s.kohaku.love/2024/06/firefox_2024-06-21_18-22-56.png)

Reply:
Icons should go before or after the title, or in the information lines at the bottom, imo. It should definitely not be shifting the layout of entire blocks.